src/index.ts

```diff
@@ -33,13 +33,24 @@ export {
   setExtras,
   setTag,
   setTags,
+  flush,
   setUser,
   // eslint-disable-next-line deprecation/deprecation
   startTransaction,
+  withScope,
+
+  // V8
+  startInactiveSpan,
   startSpan,
   startSpanManual,
-  startInactiveSpan,
-  withScope,
+  getActiveSpan,
+  spanToJSON,
+  spanIsSampled,
+  setMeasurement,
+  getGlobalScope,
+  // getIsolationScope, TODO: Verify why it's not working
```

Comment:
Could you share more details what is not working?

Reply:
It wasn't finding the client so any operation done on the isolated scope was lost.